agent/src/main/java/com/code_intelligence/jazzer/runtime/FuzzTargetFinder.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.code_intelligence.jazzer.runtime
+
+import java.util.jar.Manifest
+
+@Suppress("unused")
+internal object FuzzTargetFinder {
+
+    /**
+     * Returns the value of the `Fuzz-Target-Class` manifest attribute if there is a unique one among all manifest
+     * files in the classpath.
+     */
+    @JvmStatic
+    fun detectFuzzTargetClass(): String? {
+        val manifests = FuzzTargetFinder::class.java.classLoader.getResources("META-INF/MANIFEST.MF")
+        val fuzzTargets = manifests.asSequence().mapNotNull { url ->
+            url.openStream().use { inputStream ->
+                val manifest = Manifest(inputStream)
+                manifest.mainAttributes.getValue("Fuzz-Target-Class")
+            }
+        }.toList()
+        return if (fuzzTargets.size == 1) fuzzTargets.first() else null
```

Comment:
If two jar files with this manifest entry end up in the class path the user gets no indication that this manifest entry is being ignored. Do you think it makes sense to log a simple warning message in that case?

Reply:
I added a warning.

Reply:
Yes, I think the warning would be enough. Thanks!